kjourney/src/app/bands/bands.component.html

```diff
@@ -0,0 +1,58 @@
+<router-outlet></router-outlet>
+<header>
+  <table style="width: 100%;">
+    <tr>
+      <th>
+</th>
+<th>
+</th>
+</tr>
+</table>
+</header>
+
+<div class="topnav">
+  <a href="#">Your Role</a>
+  <a href="#">Support</a>
+</div>
+
+<section>
+  <article>
+  <table id="competency" style="width:100%">
+		<thead>
+			<tr>
+				<th>BandID</th>
```

Comment:
Why is the band ID being shown here? Is this literally the primary key from the table?